pkg/features/antrea_features.go

```diff
@@ -50,6 +50,10 @@ const (
 	// alpha: v0.10
 	// Enable collecting and exposing NetworkPolicy statistics.
 	NetworkPolicyStats featuregate.Feature = "NetworkPolicyStats"
+
+	// alpha: v0.1
```

Comment:
v0.1 -> v0.11 if the expectation is that it will be included in the upcoming release
but maybe this is TBD and the release is only about 10 days away

Reply:
I think it's safe to set this to `v0.12`